package.json

```diff
@@ -26,8 +30,7 @@
     "web-ext-run:firefox": "npm run web-ext-run -- --target firefox-desktop",
     "web-ext-run:firefox-mv3": "npm run web-ext-run -- --firefox-preview --pref=extensions.eventPages.enabled=true --verbose",
     "web-ext-run:android": "npm run web-ext-run -- --target firefox-android --adb-device $npm_config_device --firefox-apk org.mozilla.fenix",
-    "web-ext-run:chrome": "npm run web-ext-run -- --target chromium",
-    "web-ext-run:desktop": "npm run web-ext-run -- --target chromium --target firefox-desktop"
```

Comment:
Removed this npm script as it will the add-on can no longer work simultaneously.